modules/extensions/partials/proc_creating-a-telemetry-plugin.adoc

```diff
@@ -86,7 +86,7 @@ include::example$telemetry/pom_snippet.xml[]
 
 . Add the Apache HTTP components library to send HTTP requests.
 
-. Consult the link:https://github.com/che-incubator/che-workspace-telemetry-client/packages[GitHub packages] for the latest version and Maven coordinates of `back-end-base`. link:https://help.github.com/en/packages/publishing-and-managing-packages/about-github-packages[GitHub packages] require a personal access token with `read:packages` permissions to download the {prod-short} telemetry libraries. Create a personal access token and copy the token value.
```

Comment:
"latest version" was removed since the more recent versions of the dependency are specific to devworkspaces.